test/run.py

```diff
@@ -227,6 +243,14 @@ def test_qa_regex(self):
         ec, output = run_qas([sys.executable, SCRIPT_QA, 'whattime'], qa_reg=qa_dict)
         self.assertEqual(ec, 0)
 
+    def test_qa_legacy_regex(self):
+        """Test regex based q and a (works only for qa_reg)"""
+        qa_dict = {
+                   '\s(?P<time>\d+(?:\.\d+)?).*?What time is it\?': '%(time)s',
+                   }
```

Comment:
I don't see the point in spreading this across multiple lines?

And if you insist, just indent 2nd line with 4 spaces compared to `qa_dict = {`?

Reply:
it tests something else, i'd rathe rnot mix it up
i'll reindent